source/go-perun/new_tutorial/main/setup.rst

```diff
@@ -0,0 +1,173 @@
+.. _setupscenario:
+
+Setup Scenario
+##############
+Before creating our exemplary scenario, we construct a few helper functions that will cover the contract deployment, Client generation, and logging of balances in the command line.
+We put the code of this section into `util.go`
+
+Deploy Contracts
+================
+
+go-perun uses two on-chain contracts: the Adjudicator and the Asset Holder.
+They are written in the contract language of Ethereum; solidity.
+
+Each contract must be deployed before go-perun can be used.
+Usually, you would assume that they are already deployed, and the addresses are known in advance.
+But since this is a complete example for a local chain, we must deploy them.
+
+Concept: Adjudicator  & Asset Holder
+------------------------------------
+The Adjudicator contract ensures that a user can consistently enforce the rules of his Channel.
+Since the central part of the communication is off-chain, he would only contact the Adjudicator if he felt betrayed by the participants.
```

Comment:
```suggestion
Since the central part of the communication is off-chain, he would only contact the Adjudicator if he felt betrayed by the participants or for concluding the channel.
```